sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_content_safety/_self_harm.py

```diff
@@ -43,8 +42,8 @@ class SelfHarmEvaluator(RaiServiceEvaluatorBase):
     @override
     def __init__(
         self,
+        credential,
```

Comment:
Curious why we change the order?

Reply:
in Azure SDKs, the credential appears as the first positional arg in the client constructor, so just aligning to that pattern here. Moving from keyword to positional is anyway a breaking change, so figured we can align to the Azure SDK standard when we can.

Reply:
Thanks for the info. Let's follow with other Azure SDK.